component/site/views/default/helpers/defaultloadedfromtemplate.php

```diff
@@ -881,7 +881,7 @@ function DefaultLoadedFromTemplate($view, $template_name, $event, $mask, $templa
 
 								$row = $event;
 								$times = "";
-								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat" || $jevtask == "range")
+								if (($showyeardate && $jevtask == "year") || $jevtask == "search.results" || $jevtask == "month.calendar" || $jevtask == "cat")
```

Comment:
I've merged so many times it's lost the original pull naming. 

Reasoning behind this is uniformity as requested b @brianteeman Full commit to this one here: https://github.com/JEvents/JEvents/pull/361/commits/407d7ab1639bc1852d383295126f5f5bc11243c5

Range views now always have a date column to the left. So tend not toneed the date output again. Any users requiring the date and do so easily with the additional date string.